core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -8110,6 +8187,7 @@ components:
           # NOTE: These need to match LedgerTransactionType
           Genesis: '#/components/schemas/GenesisLedgerTransaction'
           User: '#/components/schemas/UserLedgerTransaction'
+          UserV2: '#/components/schemas/UserLedgerTransactionV2'
```

Comment:
UserV2 is not added to LedgerTransactionType, it fails to generate proper models.

Reply:
Thanks for the heads up, this is fixed in the latest push 👍

Reply:
Fixed in latest push, thanks for spotting this